t3/main.py

```diff
@@ -107,7 +106,7 @@ class T3(object):
         rmg (dict): RMG directives.
         qm (dict, optional): QM directive.
         verbose (int): The logging level.
-        t0 (float): Initial time when the project was spawned.
+        t0 (float): Initial time when the project was spawned, stored as a datetime object.
```

Comment:
Super minor but I had forgotten to update this docstring from float to datetime.datetime. I'll push a minor fix.